plugins/sentinelone/plugin.spec.yaml

```diff
@@ -2088,40 +2103,39 @@ actions:
   quarantine:
     title: Quarantine
-    description: Isolate (quarantine) endpoint from the network
+    description: Isolate (quarantine) endpoints from the network
     input:
       agent:
         title: Agent
         description: Agent to perform quarantine action on. Accepts IP address, MAC address, hostname, UUID or agent ID
-        type: string
-        example: hostname123
+        type: "[]string"
+        example: '["hostname123", "hostname456"]'
         required: true
       quarantine_state:
         title: Quarantine State
-        description: True to quarantine host, false to unquarantine host
+        description: True to quarantine host, false to unquarantine hosts
         type: boolean
         example: true
         required: true
       whitelist:
         title: Whitelist
         description: This list contains a set of devices that should not be blocked. This can include IPs, hostnames, UUIDs and agent IDs
         type: "[]string"
-        example: ["198.51.100.100", "hostname123", "901345720792880606", "28db47168fa54f89aeed99769ac8d4dc"]
+        example: '["198.51.100.100", "hostname123", "901345720792880606", "28db47168fa54f89aeed99769ac8d4dc"]'
         required: false
-      case_sensitive:
-        title: Case Sensitive
-        description: Looks up the specified Agent in a case-sensitive manner. Setting this value to false may result in longer run times and unintended results
-        type: boolean
-        example: true
-        default: true
-        required: true
     output:
-      response:
-        title: Response
-        description: SentinelOne API call response data
-        type: quarantine_response
-        required: false
-        example: '{ "response": { "data": [ { "agentDomain": "WORKGROUP", "agentGroupId": "123123456712356789", "agentId": "123123456712356789" } } }'
+      successful:
+        title: Successful
+        description: List containing hosts with successful operation
+        type: "[]string"
+        required: true
+        example: '["hostname123", "hostname456"]'
+      failures:
+        title: Failure
+        description: List containing hosts with failure operation
```

Comment:
Suggested reword:
List of hosts on which the operation attempt was unsuccessful.